src/VsixTesting.Invoker/InvokerPackage.cs

```diff
@@ -5,13 +5,12 @@ namespace VsixTesting.Invoker
     using System;
     using Microsoft.VisualStudio;
     using Microsoft.VisualStudio.Shell;
+    using EnvDTE;
 
     [PackageRegistration(UseManagedResourcesOnly = true)]
     [ProvideAutomationObject(AutomationObjectName)]
-#if DEBUG
```

Comment:
Are you sure `ProvideAutoLoad` is necessary? Won't the package get loaded before `GetAutomationObject` is called anyway?